packages/devtools_app/lib/src/screens/vm_developer/object_inspector_view.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../shared/split.dart';
+import '../debugger/program_explorer.dart';
+import '../debugger/program_explorer_controller.dart';
+import '../debugger/program_explorer_model.dart';
+import 'object_viewport.dart';
+import 'vm_developer_tools_screen.dart';
+
+/// Displays a program explorer used to select and display information about objects in the Dart VM
```

Comment:
Nit: this should probably be a two line comment (I'm pretty sure it's > 80 characters long).